site/classes/output.class.php

```diff
@@ -990,6 +990,65 @@ static function flyer($data, $image, $type, $id = null)
 
 
 	/**
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
```

Comment:
strange, I did not made this: Now are only the changed raws in! (is this made automaticly?